src/api/src/helper.rs

```diff
@@ -404,6 +441,10 @@ pub fn pb_value_to_value_ref(value: &v1::Value) -> ValueRef {
             let interval = Interval::from_month_day_nano(v.months, v.days, v.nanoseconds);
             ValueRef::Interval(interval)
         }
+        ValueData::DurSecondValue(v) => ValueRef::Duration(Duration::new_second(*v)),
```

Comment:
I think using the complete `Duration` is better than `Dur` prefix. What's your opinion?  @waynexia